notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<p>Is the bottom ref file for the STIS correct? I couldn't find hst_cos_0368.imap on this page, but I did find one for STIS:&nbsp;<a href="https://hst-crds.stsci.edu" target="_blank">https://hst-crds.stsci.edu/</a>&nbsp;hst_stis_0368</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>